src/com/mateacademy/factory/Car.java

```diff
@@ -0,0 +1,135 @@
+package com.mateacademy.factory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private final int PRODUCTION_DATE;
```

Comment:
PRODUCTION_DATE such naming is used only for constants. In your case, it is not constant.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.5-non-constant-field-names
Non-constant field names (static or otherwise) are written in lowerCamelCase.
And please use LocalDate type for productDate.
private final LocalDate productionDate;